roles/agent_install/tasks/main.yml

```diff
@@ -76,12 +76,11 @@
           when: agent_install_probe_loaded.stdout == 'LOADED'
           changed_when: agent_install_probe_removed.rc == 0
 
-    - name: (kmodule) Disable eBPF
-      ansible.builtin.lineinfile:
-        path: /etc/sysconfig/dragent
-        regexp: SYSDIG_BPF_PROBE
+    - name: (kmod) Disable eBPF
+      ansible.builtin.file:
+        path: "/etc/{% if ansible_pkg_mgr == 'apt' %}default{% else %}sysconfig{% endif %}/dragent"
```

Comment:
@mprzybylski do you know why this file is indifferent location depending from the package manger? 🤔

Reply:
@mavimo, the way I understand it is different distribution families following slightly different layout conventions.